test/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolProjectWithPackagedShim.cs

```diff
@@ -157,7 +157,7 @@ public void It_uses_customized_PackagedShimOutputRootDirectory(bool multiTarget,
 
             var packCommand = new PackCommand(Log, helloWorldAsset.TestRoot);
 
-            packCommand.Execute().Should().Pass();
+            packCommand.Execute("-bl:{}").Should().Pass();
```

Comment:
Is this trying to create a binlog during test execution? As mentioned in our triage, even if it works, you cannot get at the file directly in Helix. We need a proper way to do this.

Reply:
You can hard code the binlong path in the -bl command, and then dump the binary content into the test output, copy paste it into a file to read it. I like the idea of properly dumping the log from helix, but I don't want to block this PR to do so

Reply:
I think I've figured it out, but now Helix is busted. In either case I'll be removing this before we merge this PR.